appveyor.yml

```diff
@@ -15,7 +15,7 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   # install modules
-  - npm install --msvs_version=2013
+  - npm install --msvs_version=2015
```

Comment:
Will this still build with msvs_version set to 2013?

Reply:
let us use the 2013 compiler :)